speed-dreams/source-2.2.3/data/data/indicators/config.xml

```diff
@@ -1,83 +1,168 @@
-            <section name="texture">
-                <attstr name ="source" val="Intervention_Steer_Left_Proto_256.png"/>
-                <attnum name ="xpos"   val="0.75"/>
-                <attnum name ="ypos"   val="0.03"/>
-            </section>
-            <section name="text">
-                <attstr name ="content"   val="Steer left"/>
-                <attnum name ="xpos"      val="0.75"/>
-                <attnum name ="ypos"      val="0"/>
-                <attstr name ="font"      val="jura-normal.glf"/>
-                <attnum name ="font-size" val="300"/>
-            </section>
-            <section name="sound">
-                <attstr name ="source" val="warning_left_driftingright.wav"/>
-                <attstr name="looping" val="yes"/>
-                <attnum name="loop-interval" val="3"/>
-            </section>
-        </section>
-        <section name ="steer right">
-            <section name="texture">
-                <attstr name ="source" val="Intervention_Steer_Right_Proto_256.png"/>
-                <attnum name ="xpos"   val="0.75"/>
-                <attnum name ="ypos"   val="0.03"/>
-            </section>
-            <section name="text">
-                <attstr name ="content"   val="Steer right"/>
-                <attnum name ="xpos"      val="0.75"/>
-                <attnum name ="ypos"      val="0"/>
-                <attstr name ="font"      val="jura-normal.glf"/>
-                <attnum name ="font-size" val="300"/>
-            </section>
-            <section name="sound">
-                <attstr name ="source" val="warning_right_driftingleft.wav"/>
-                <attstr name="looping" val="no"/>
-            </section>
-        </section>
-        <section name ="brake">
-            <section name="texture">
-                <attstr name ="source" val="Intervention_Brake_Proto_256.png"/>
-                <attnum name ="xpos"   val="0.75"/>
-                <attnum name ="ypos"   val="0.03"/>
-            </section>
-            <section name="text">
-                <attstr name ="content"   val="Brake"/>
-                <attnum name ="xpos"      val="0.75"/>
-                <attnum name ="ypos"      val="0"/>
-                <attstr name ="font"      val="jura-normal.glf"/>
-                <attnum name ="font-size" val="300"/>
-            </section>
-            <section name="sound">
-                <attstr name ="source" val="warning_brake_brake.wav"/>
-            </section>
-        </section>
-        <section name ="accelerate">
-            <section name="text">
-                <attstr name ="content"   val="Accel"/>
-                <attnum name ="xpos"      val="0.75"/>
-                <attnum name ="ypos"      val="0"/>
-                <attstr name ="font"      val="jura-normal.glf"/>
-                <attnum name ="font-size" val="300"/>
-            </section>
-         </section>
-    </section>
+	<section name="Interventions">
```

Comment:
Remove tabs from file. Also in my opinion the disticntion between the intervention type should be made at the top level of the xml (like section=shared_control, etc) instead of having texture1, texture2, etc. This also enables us to add other sounds or text for each intervention type easily, without having to name them 'sound1, sound2, etc', which is not clear at all.